docs/app-integration-development/09-OAuth-2-Auth-Code-Grant.md

```diff
@@ -94,8 +103,54 @@ Once obtained, access tokens can be used to make [REST API](https://api-referenc
 When making an API request, include the version of the API in the `Accept` header. Access tokens must also be sent in the request as part of the `Authorization` header along with the `Bearer` token type, using this format:
 
 ```http
-Authorization: Bearer 9937611c354d287d3ff509afdde5b1d6d500c73a67387d666ca1e8e3d502d516
+Authorization: Bearer pdus+_0XBPWQQ_b2b2060b-e7af-44a1-8ddf-9c56fedd8d4f
 Accept: application/vnd.pagerduty+json;version=2
 ```
 
-The token can be used continuously to make requests until the user or app owner revokes it.
+## Getting a new Access Token with a Refresh Token
+
+All of our current access tokens have an expiration date defined -- prior to October 30th 2022, access tokens lasted for a year, but after October 30th 2022, access tokens for new OAuth clients will only last for one hour. In April 2023, all new access tokens for all OAuth clients will start using the new expiry settings. So, it would be to your benefit to implement OAuth refresh.
+
+Exchanging the refresh token for the access token is similar to using an authorization code: send a POST request to the token endpoint, but using the `refresh_token` grant type instead.
+
+The body of the request should include the following parameters: `client_id`, `client_secret`, the `refresh_token` previously received from PagerDuty, and `grant_type=refresh_token`. The content type should still be `application/x-form-urlencoded`.
+
+```
+curl -X POST https://identity.pagerduty.com/oauth/token \
+  -d "grant_type=refresh_token" \
+  -d "client_id={CLIENT_ID}" \
+  -d "client_secret={CLIENT_SECRET}" \
+  -d "refresh_token=pdus+_1XBPWQQ_f85dd2f1-b906-478a-a9e6-678952529e4e"
+```
+
+A successful response will include a new access token and a new refresh token:
+
+```json
+{
+    "token_type": "bearer",
+    "access_token": "pdus+_0XBPWQQ_39435d07-9232-4bc2-9dc9-c4a8fccc94ad",
+    "refresh_token": "pdus+_1XBPWQQ_4f0dd763-8665-4fd7-aab7-d9f98e0c151b",
+    "scope": "openid write",
+    "expires_in": 864000
+}
+```
+
+## Token Expiries
+
+We've noted above that access tokens have an expiry date -- so do refresh tokens, and so does the lifetime of the user's authorization.
+
+Prior to October 30th 2022, all registered OAuth clients have the following settings:
+
+ - access token expiry of one year
+ - refresh token expiry of one year
+
+After October 30th 2022, all newly registered OAuth clients will have the following settings:
+
+ - access token expiry of one day
+ - refresh token expiry of 30 days
+ - rolling refresh window of 3 years
```

Comment:
This might be unclear to some users?

Reply:
Does this sentence help clear things up:

> Once you have implemented the refresh token flow, this will allow your customers to use your OAuth app continuously for three years, as long as they use it at least once every 30 days.

Reply:
Yep that does